### ISSUE TITLE
[abseil] Error code 0xc0000017 on x64-windows

### ISSUE THREAD

**Issue by martingalvan-volue** [microsoft/vcpkg#37213]
**Describe the bug**
When trying to build abseil on x64-windows on the latest vcpkg master, I see the following:

```
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:134 (message):

  C:/Users/martin.galvan/Documents/vcpkg/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin/pkg-config.exe
  --exists absl_str_format failed with error code: Exit code 0xc0000017



      ENV{PKG_CONFIG_PATH}: "C:/Users/martin.galvan/Documents/vcpkg/packages/abseil_x64-windows/debug/lib/pkgconfig;C:/Users/martin.galvan/Documents/vcpkg/packages/abseil_x64-windows/share/pkgconfig;C:/Users/martin.galvan/Documents/vcpkg/installed/x64-windows/debug/lib/pkgconfig;C:/Users/martin.galvan/Documents/vcpkg/installed/x64-windows/share/pkgconfig"
      output:
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:196 (z_vcpkg_fixup_pkgconfig_check_files)
  ports/abseil/portfile.cmake:34 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:172 (include)
```

The same thing may happen for other packages as well. Note that I've installed CMake and Powershell myself on this PC; if I use the vcpkg-provided ones instead, I get the same error but with a more descriptive text (`"Not enough memory resources are available to process this command"`).

This is on a 64 GB RAM PC with minimal load, which I assume should work (RAM usage is around 25% at most when this happens). I've tried setting different values for `VCPKG_MAX_CONCURRENCY`, but the error persists.

I see the failure on the latest master. Using vcpkg 2024.02.14 works fine, as do earlier versions.

**Environment**
- OS: Windows 11 x86_64
- Compiler: Visual Studio 2022

**To Reproduce**
1. vcpkg install abseil[cxx17]:x64-windows
2. See error

**Expected behavior**
Abseil builds correctly.

**Failure logs**
I'm attaching the console output. The configure and build logs are fine, and don't show anything out of the ordinary.

[stdout-x64-windows.log](https://github.com/microsoft/vcpkg/files/14522120/stdout-x64-windows.log)

**Comment by dg0yt** [comment]
> I suspect the #36151 commit caused this failure

Could you elaborate? Did you bisect or try to revert that commit?

More general: Can you set the environment variable and run the command in a command line?
Replace `--exists` with `--cflags` to get real output. There is also `--debug`. (I'm getting more familar with `pkgconf` debugging, but I don't have MSVC for local tests.)

**Comment by dg0yt** [comment]
- I don't see how the crashes could be related to #36151. According to the error message, that PR has no (and does not intent any) extra effect on the environment (`PKG_CONFIG_PATH`) when the `pkg-config` command is run, i.e. no change on observable behavior.
- I am aware of massive inefficiencies of current (2.x) versions of pkgconf, but the version is the same in 2024.02.14. (PRs pending.)
- From the msys2 packages, only autoconf was rolled back after 2024.02.14, but this doesn't seem related to `pkg-config`.

**Comment by vasil-pashov** [comment]
I am experiencing the same issue. I'm contributing to an opensource project. This after [this](https://github.com/man-group/ArcticDB/commit/75b14e14b433770a118e111ae7bcd31931797672) commit abseil builds started failing when MSVS is used. Our Windows CI uses MSVC + ninja for building and the build passes (linux builds with gcc are also passing). However locally I'm using MSVS 2022 and abseil build fail with the same error as above. Rolling back vcpkg to `836a2d684dac6b543563e53ab30a1a5b8afaf97b` worked for me.

**Comment by ptrnycz** [comment]
> I am experiencing the same issue. I'm contributing to an opensource project. After [this](https://github.com/man-group/ArcticDB/commit/75b14e14b433770a118e111ae7bcd31931797672) commit abseil builds started failing when MSVS is used. Our Windows CI uses MSVC + ninja for building and the build passes (linux builds with gcc are also passing). However locally I'm using MSVS 2022 and abseil build fail with the same error as above. Rolling back vcpkg to `836a2d684dac6b543563e53ab30a1a5b8afaf97b` worked for me.

//////////////////////////////////////////////////////////////////////////////////////////////

I am trying to compile open-source project, and I am experiencing the same issue too. Could you advice please step by step what should I do to install abseil using vcpkg without errors in Windows 10?

**Comment by Sovak** [comment]
I can confirm this, reverting to [2024.02.14 Release](https://github.com/microsoft/vcpkg/releases/tag/2024.02.14) seems to have resolved it for me. But I am getting a different error with pkg-config.exe.

"--exists absl_log_internal_conditions failed with error code: not enough memory resources are available to process this command"

**Comment by dg0yt** [comment]
Let's hope pkgconf 2.2 is released and available in msys2 soon. All previous versions mean trouble: Going to be removed from msys2 soon, producing wrong results, or exploding over abseil.

**Comment by dg0yt** [comment]
You may test #37844.

**Comment by VelocityRa** [comment]
I also still get the push error, but then a success message (caches are actually uploaded).

**Comment by dg0yt** [comment]
> The problem persists even after updating to the latest commit

That is really sad. Can you re-open the issue?

Thanks for the project link. I forked it, and see the same error. I will try to see if I can find out more, as time permits.
It might be a different problem, such as a msys runtime mismatch.

Another observation is that the CMake configuration continues after manifest mode install failure, i.e. without a full set of dependencies. I don't think this is intented behavior. A project might silently disable features or pick system libraries when the vcpkg package is missing.

**Comment by Pravila00** [comment]
@dg0yt do we have any news on this? I am using `1751f9f8c732c2e6f9e81ce56c10e4c4aa265b4a` but it doesn't fix the issue.

I am facing this issue locally which is blocking my team:
```
vcpkg/downloads/tools/msys2/d33df895d441a36d/mingw64/bin/pkg-config.exe
  --exists absl_log_internal_voidify failed with error code: Exit code
  0xc0000017
```

Another member of my team is seeing this issue:
```
vcpkg/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin/pkg-config.exe
  --exists absl_log_internal_config failed with error code: Not enough memory
  resources are available to process this command
```

**Comment by Pravila00** [comment]
Could I ask you prioritize more this issue? Thanks!

**Comment by dg0yt** [comment]
> Could I ask you prioritize more this issue? Thanks!

If "you" is me: I'm only a community member.

**Comment by Pravila00** [comment]
@dg0yt sure, I was just referring to the Vcpkg team at Microsoft :)

**Comment by dg0yt** [comment]
> do we have any news on this?

What I found earlier this morning is that your GH actions build is indeed facing a problem with mixing different mingw64 runtimes (as mentioned in https://github.com/microsoft/vcpkg/issues/37213#issuecomment-2033583375).

I can fix it for abseil in `vcpkg_fixup_pkgconfig`:
https://github.com/dg0yt/vcpkg/commit/75043b7c4eae8bf0ff28432c3409779300ff4172,  
https://github.com/dg0yt/proton-bridge/actions/runs/8700512538,  
but ports might still fail for other uses of PKGCONFIG.

This path when calling vcpkg - note Windows git mingw64 dirs:
~~~
C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\runneradmin\bin;C:\Program Files\PowerShell\7;C:\Users\runneradmin\go\bin;C:\hostedtoolcache\windows\go\1.21.6\x64\bin;D:\a\proton-bridge\Qt\6.4.3\msvc2019_64\bin;C:\Users\runneradmin\AppData\Roaming\Python\Python311\Scripts;C:\hostedtoolcache\windows\Python\3.11.9\x64\Scripts;C:\hostedtoolcache\windows\Python\3.11.9\x64;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS;C:\tools\zstd;C:\Program Files\Mercurial;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\usr\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.402-6\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files\dotnet;C:\Program Files\PowerShell\7;C:\Program Files\Microsoft\Web Platform Installer;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn;C:\Program Files\Microsoft SQL Server\150\Tools\Binn;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn;C:\Program Files\Microsoft SQL Server\140\DTS\Binn;C:\Program Files\Microsoft SQL Server\150\DTS\Binn;C:\Program Files\Microsoft SQL Server\160\DTS\Binn;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI;C:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2;C:\Program Files\Amazon\SessionManagerPlugin\bin;C:\Program Files\Amazon\AWSSAMCLI\bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps
~~~

But... the vcpkg tool is to clean the `PATH` before building ports on Windows. The next step is to track down how the PATH is modified inside vcpkg's cmake script code. (It didn't find anything obvious in port abseil, and this is strange.)

**Comment by dg0yt** [comment]
And this is `PATH` where my modifications` now takes the backup:
~~~
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.39.33519\bin\HostX64\x64
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\VCPackages
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin\Roslyn
C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\
C:\Program Files (x86)\HTML Help Workshop
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\DiagnosticsHub\Collector
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console
C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64
C:\Program Files (x86)\Windows Kits\10\bin\\x64
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\\MSBuild\Current\Bin\amd64
C:\Windows\Microsoft.NET\Framework64\v4.0.30319
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\
C:\Program Files\PowerShell\7
C:\Windows\system32
C:\Windows
C:\Windows\system32\Wbem
C:\Windows\system32\WindowsPowerShell\v1.0\
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\Llvm\x64\bin
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\Linux\bin\ConnectionManagerExe
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\vcpkg
C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja
~~~
No obvious traces of another mingw/msys runtime. And still, prepending  
`D:/a/proton-bridge/proton-bridge/extern/vcpkg/downloads/tools/msys2/d33df895d441a36d/mingw64/bin`  
makes a difference for running  
`D:/a/proton-bridge/proton-bridge/extern/vcpkg/downloads/tools/msys2/d33df895d441a36d/mingw64/bin/pkg-config.exe`.

**Comment by dg0yt** [comment]
After switching computers: There is no dynamic msys/mingw runtime involved here. The `pkg-config.exe` wants:
~~~
        DLL Name: KERNEL32.dll
        DLL Name: msvcrt.dll
        DLL Name: libpkgconf-5.dll
~~~
Only `libpkgconf-5.dll` is in the same directory. It has:
~~~
        DLL Name: ADVAPI32.dll
        DLL Name: KERNEL32.dll
        DLL Name: msvcrt.dll
~~~
Shouldn't it always use the DLL next to the EXE (7) before looking at PATH (12, last!)?
https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order#search-order-for-unpackaged-apps
So how can changing PATH change behavior? :thinking:

**Comment by Pravila00** [comment]
Thanks for reporting here your investigation so far @dg0yt 

I have done some local testing, and I can confirm this commit https://github.com/microsoft/vcpkg/pull/36151 is the one breaking the abseil build. The issue of breaking abseil is that gRPC build is broken as well, which in my team is a huge dependency for several other projects.

I'd suggest reverting the commit https://github.com/microsoft/vcpkg/pull/36151, and continue investigating, otherwise our development is broken as we depend on some other libraries that were updated/added after https://github.com/microsoft/vcpkg/pull/36151

**Comment by Neumann-A** [comment]
So stupid question: From which shell is vcpkg run here?
I cannot reproduce the error....

**Comment by Pravila00** [comment]
> So stupid question: From which shell is vcpkg run here? I cannot reproduce the error....

I am using PowerShell 7.2.18 in my dev machine.

**Comment by Neumann-A** [comment]
And I don't see it Powershell or CMD or Git Bash. So what is the requirement to reproduce this?

**Comment by dg0yt** [comment]
> I have done some local testing, and I can confirm this commit #36151 is the one breaking the abseil build.

Did you roll back all of vcpkg, or did you revert the commit.

What I did before https://github.com/microsoft/vcpkg/issues/37213#issuecomment-2058681071 was to revert only the change to `vcpkg_fixup_pkgconfig`,  and this did not resolve the error.
https://github.com/dg0yt/proton-bridge/actions/runs/8684478253 

Summary:
- PKG_CONFIG_PATH is the same in all tests.
- `pkg-config.exe` (which is actually MSYS2 pkgconf) fails depending on `PATH`.

**Comment by dg0yt** [comment]
> Why can't we just revert the offending commit first, thus unblocking anyone who wishes to use vcpkg master, and then investigate and figure out the root cause of the issue?

Again: Did anyone verify that *reverting* the commit fixes the issue?
And the commit fixed other issues. So it may not be as simple as "revert".

**Comment by Pravila00** [comment]
> > I have done some local testing, and I can confirm this commit #36151 is the one breaking the abseil build.
> 
> Did you roll back all of vcpkg, or did you revert the commit?
> 
> What I did before [#37213 (comment)](https://github.com/microsoft/vcpkg/issues/37213#issuecomment-2058681071) was to revert only the change to `vcpkg_fixup_pkgconfig`, and this did not resolve the error. https://github.com/dg0yt/proton-bridge/actions/runs/8684478253
> 
> Summary:
> 
> * PKG_CONFIG_PATH is the same in all tests.
> * `pkg-config.exe` (which is actually MSYS2 pkgconf) fails depending on `PATH`.

Let me test reverting the commit, I've tried building abseil using just the previous commit of https://github.com/microsoft/vcpkg/pull/36151 where abseil build worked, and then I updated to https://github.com/microsoft/vcpkg/pull/36151, where abseil build stopped working.

I am going to revert the commit locally, and I will let you know.

**Comment by Neumann-A** [comment]
Also this might be an issue:
```
E:/vcpkg_folders/paraview.very.long.path.test.with.user/scripts/cmake/vcpkg_host_path_list.cmake(64):  set(ENV{PATH} C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.39.33519\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\VC\VCPackages;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\bin\Roslyn;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools;C:\Program Files\Microsoft Visual Studio\2022\Community\Team Tools\DiagnosticsHub\Collector;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files\Microsoft Visual Studio\2022\Community\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\;C:\Program Files\PowerShell\7;C:\Windows\system32;C:\Windows;C:\Windows\system32\Wbem;C:\Windows\system32\WindowsPowerShell\v1.0\;C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\Llvm\x64\bin;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\VC\Linux\bin\ConnectionManagerExe;C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin;E:/vcpkg_folders/paraview.very.long.path.test.with.user/downloads/tools/msys2/fdbea3694fb5c0d4/mingw64/bin )
```

**Comment by Neumann-A** [comment]
https://github.com/microsoft/vcpkg/blob/501cb01e517ee5689577bb01ba8bd1b4c1041a53/scripts/cmake/z_vcpkg_setup_pkgconfig_path.cmake#L21

Is probably making PATH too long since it adds the path for every invoke of the function.

**Comment by Pravila00** [comment]
> > > I have done some local testing, and I can confirm this commit #36151 is the one breaking the abseil build.
> > 
> > 
> > Did you roll back all of vcpkg, or did you revert the commit?
> > What I did before [#37213 (comment)](https://github.com/microsoft/vcpkg/issues/37213#issuecomment-2058681071) was to revert only the change to `vcpkg_fixup_pkgconfig`, and this did not resolve the error. https://github.com/dg0yt/proton-bridge/actions/runs/8684478253
> > Summary:
> > 
> > * PKG_CONFIG_PATH is the same in all tests.
> > * `pkg-config.exe` (which is actually MSYS2 pkgconf) fails depending on `PATH`.
> 
> Let me test reverting the commit, I've tried building abseil using just the previous commit of #36151 where abseil build worked, and then I updated to #36151, where abseil build stopped working.
> 
> I am going to revert the commit locally, and I will let you know.

Reverting the commit `215a2535590f1f63788ac9bd2ed58ad15e6afdff` makes abseil to build again.

**Comment by Pravila00** [comment]
> https://github.com/microsoft/vcpkg/blob/501cb01e517ee5689577bb01ba8bd1b4c1041a53/scripts/cmake/z_vcpkg_setup_pkgconfig_path.cmake#L21
> 
> Is probably making PATH too long since it adds the path for every invoke of the function.

This makes it work for me too, it seems the issue depends on the length of the PATH where vcpkg is installing abseil.

**Comment by Neumann-A** [comment]
> This makes it work for me too, it seems the issue depends on the length of the PATH where vcpkg is installing abseil.

No the problem is that the PATH to pkg-config gets added N times for every pc file in abseil leading to an overflow of PATH

**Comment by Pravila00** [comment]
Thanks for the investigation @Neumann-A !

So now that we have the cause of why this is happening, do we know how we can solve this issue?

**Comment by dg0yt** [comment]
So it is the side effect on `PATH` which is not restored by `z_vcpkg_restore_pkgconfig_path`. 
And we even don't need the change to `PATH` for `vcpkg_fixup_pkgconfig`, only `PKG_CONFIG_PATH`. 😢

**Comment by Filyus** [comment]
There probably another issue with PATH, I tried even the 2024.02.14 version, there error too. Short VCPKG_DOWNLOADS helps.
